demo-project/package.json

```diff
@@ -1,26 +1,28 @@
 {
-    "name": "demo-project",
-    "version": "1.0.0",
-    "main": "index.js",
-    "license": "MIT",
-    "scripts": {
-        "build": "parcel build main.ts"
-    },
-    "dependencies": {
-        "body-parser": "^1.20.2",
-        "cookie-parser": "^1.4.6",
-        "cors": "^2.8.5",
-        "express": "^4.18.2",
-        "express-jwt": "^8.4.1",
-        "jsonwebtoken": "^9.0.2",
-        "qs": "^6.12.1"
-    },
-    "devDependencies": {
-        "@types/cookie-parser": "^1.4.3",
-        "@types/cookie-session": "^2.0.44",
-        "@types/cors": "^2.8.13",
-        "@types/express": "^4.17.17",
-        "@types/express-jwt": "^7.4.2",
-        "parcel": "^2.9.3"
-    }
+  "name": "demo-project",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "parcel build main.ts",
+    "format": "npx prettier --write ."
```

Comment:
We also need another script to check for lint errors in the CI, as we do here: https://github.com/KILTprotocol/sdk-js/blob/23c5b045dc7d3d587aeca5f6650736bb1218130b/.github/workflows/tests.yml#L73.